pkg/api/pingpong.go

```diff
@@ -22,7 +23,12 @@ func (s *server) pingpongHandler(w http.ResponseWriter, r *http.Request) {
 	peerID := mux.Vars(r)["peer-id"]
 	ctx := r.Context()
 
-	rtt, err := s.Pingpong.Ping(ctx, peerID, "hey", "there", ",", "how are", "you", "?")
+	address, err := swarm.NewAddress(peerID)
+	if err != nil {
+		jsonhttp.NotFound(w, "peer not found")
```

Comment:
- Missing return after the response.
- Log error message as debug.
- Maybe it is better to return message `invalid peer address`